madgiraffestudios/src/components/About/About.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import Nav from 'react-bootstrap/Nav';
+import Navbar from 'react-bootstrap/Navbar';
+import giraffeicon from '../../Images/icon.png';
+import photoshoots from '../../Images/undraw_fashion_photoshoot_mtq8.png'
+import graphicdesign from '../../Images/undraw_logo_design.png'
+import uxDesin from '../../Images/undraw_mobile_ux.png'
+const About = () => (
+    <div class="container">
+        <div class=" text-center pt-5 mx-auto w-50 ">
+            <h1 class="font-weight-bold p-3">What I Do</h1>
+            <p class="font-weight-600">text pitch
+            </p><p>text pitch2</p><h6 class="pb-5"><span class="font-weight-bold">Motto:</span> motto</h6></div>
+
+           
+         <div class="row"><div class="col-md-6" id="first">
+             <img src={photoshoots} alt="goal" class="img-fluid"/>
+             </div>
+             <div class="col-md-6 mt-5 p-3" id="second">
+                 <h5 class="font-weight-bold">PhotoShoots</h5><p class="w-75 text-muted">At TotalKnockoutFitness we are dedicated to helping you achieve your personal goals. With our 24/7 online community, we seek to provide you with not only the support but the accountability you need to stay disciplined and committed to your goals.</p></div><div></div>
```

Comment:
I think we should have a look at this together, to explain what's going on here. Also, the text should be removed as it's not relative. Consider replacing this with lorem ipsum. Also, this component shouldn't be pushed until the actual content is available